devicetypes/krlaframboise/eva-logik-in-wall-smart-switch.src/eva-logik-in-wall-smart-switch.groovy

```diff
@@ -0,0 +1,634 @@
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+*/
+
+import groovy.json.JsonOutput
+import groovy.transform.Field
+
+@Field static Map commandClassVersions = [
+	0x20: 1,	// Basic
+	0x25: 1,	// Switch Binary
+	0x55: 1,	// Transport Service
+	0x59: 1,	// AssociationGrpInfo
+	0x5A: 1,	// DeviceResetLocally
+	0x5B: 1,	// CentralScene (3)
+	0x5E: 2,	// ZwaveplusInfo
+	0x6C: 1,	// Supervision
+	0x70: 1,	// Configuration
+	0x7A: 2,	// FirmwareUpdateMd
+	0x72: 2,	// ManufacturerSpecific
+	0x73: 1,	// Powerlevel
+	0x85: 2,	// Association
+	0x86: 1,	// Version (2)
+	0x8E: 2,	// Multi Channel Association
+	0x98: 1,	// Security S0
+	0x9F: 1		// Security S2
+]
+
+@Field static Map paddleControlOptions = [0:"Normal", 1:"Reverse", 2:"Toggle"]
+@Field static Integer reversePaddle = 1
+@Field static Integer togglePaddle = 2
+
+@Field static Map ledModeOptions = [0:"Off When On", 1:"On When On", 2:"Always Off", 3:"Always On"]
+
+@Field static Map associationReportsOptions = [0:"None", 1:"Physical", 2:"3-way", 3:"3-way and Physical", 4:"Digital", 5:"Digital and Physical", 6:"Digital and 3-way", 7:"Digital, Physical, and 3-way", 8:"Timer", 9:"Timer and Physical", 10:"Timer and 3-way", 11:"Timer, 3-Way, and Physical", 12:"Timer and Digital", 13:"Timer, Digital, and Physical", 14:"Timer, Digital, and 3-way", 15:"All"]
+
+@Field static Map autoOnOffIntervalOptions = [0:"Disabled", 1:"1 Minute", 2:"2 Minutes", 3:"3 Minutes", 4:"4 Minutes", 5:"5 Minutes", 6:"6 Minutes", 7:"7 Minutes", 8:"8 Minutes", 9:"9 Minutes", 10:"10 Minutes", 15:"15 Minutes", 20:"20 Minutes", 25:"25 Minutes", 30:"30 Minutes", 45:"45 Minutes", 60:"1 Hour", 120:"2 Hours", 180:"3 Hours", 240:"4 Hours", 300:"5 Hours", 360:"6 Hours", 420:"7 Hours", 480:"8 Hours", 540:"9 Hours", 600:"10 Hours", 720:"12 Hours", 1080:"18 Hours", 1440:"1 Day", 2880:"2 Days", 4320:"3 Days", 5760:"4 Days", 7200:"5 Days", 8640:"6 Days", 10080:"1 Week", 20160:"2 Weeks", 30240:"3 Weeks", 40320:"4 Weeks", 50400:"5 Weeks", 60480:"6 Weeks"]
+
+@Field static Map powerFailureRecoveryOptions = [0:"Turn Off", 1:"Turn On", 2:"Restore Last State"]
+
+@Field static Map noYesOptions = [0:"No", 1:"Yes"]
+
+metadata {
+	definition (
+		name: "EVA LOGIK In-Wall Smart Switch",
+		namespace: "krlaframboise",
+		author: "Kevin LaFramboise",
+		vid:"generic-switch",
+		ocfDeviceType: "oic.d.switch"
+	) {
+		capability "Actuator"
+		capability "Sensor"
+		capability "Light"		
+		capability "Switch"		
+		capability "Refresh"
+		capability "Health Check"
+		capability "Configuration"
+
+		attribute "firmwareVersion", "string"
+		attribute "lastCheckIn", "string"
+		attribute "syncStatus", "string"
+		
+		fingerprint mfr: "0312", prod: "FF00", model: "FF03", deviceJoinName: "Minoston In-Wall Switch" // MS10Z		
+        fingerprint mfr: "0312", prod: "A000", model: "A005", deviceJoinName: "EVA LOGIK In-Wall Switch" // ZW30
```

Comment:
Please add a Raw Description in a comment after the fingerprint so it would look like, for example:
fingerprint mfr: "0312", prod: "A000", model: "A005", deviceJoinName "EVA LOGIK Switch" // Raw Description (paste the raw description here)

Reply:
Please add a product name / model in a comment after the fingerprint (I mean the model from Developer Workspace request -> there are requests for "ZW30TS" and "ZW30S")